app/forms/schools/form.rb

```diff
@@ -35,6 +35,11 @@ class Form < TraineeForm
               },
               if: -> { no_results_searching_again? }
 
+    def initialize(trainee, **kwargs)
+      @non_search_validation = kwargs.delete(:non_search_validation)
```

Comment:
Could we initialise this in a similar manner to by adding them to the `NON_TRAINEE_FIELDS`?
https://github.com/DFE-Digital/register-trainee-teachers/pull/1209/files#diff-6ff372d10d90b172435e494ac165e8d21d8a5011531fd74a16ef3336df0d9afcL5-L10